src/nginx/t/backend_routing_constant_address.t

```diff
@@ -223,7 +223,7 @@ sub bookstore {
     or die "Can't create test server socket: $!\n";
   local $SIG{PIPE} = 'IGNORE';
 
-  $server->on('GET', '/listShelves', <<'EOF');
+  $server->on('GET', '/listShelves?site=space%20plus%2B2U%3D', <<'EOF');
 HTTP/1.1 200 OK
```

Comment:
QQ:  without the PR, what does the backend get for this escaped characters based on current implementation?

Reply:
My understanding is: for url binding parameters and query parameters, if there is backend routing, we should remain the escape characters for both. Right?

Reply:
Before this change:  in CONST_ADDRESS mode,  the backend path is un-escaped.  It will be
/listShelves?site=space plus+2U/.   Since it is un-escaped,  it fails to reach the backend.  That is the bug.